db/schema.rb

```diff
@@ -111,6 +111,7 @@
     t.datetime "establishment_processed_at", comment: "Timestamp for when the establishment has succeeded in processing."
     t.datetime "establishment_submitted_at", comment: "Timestamp for when the the intake was submitted for asynchronous processing."
     t.boolean "filed_by_va_gov", comment: "Indicates whether or not this form came from VA.gov"
+    t.boolean "homelessness", default: false, null: false, comment: "whether a veteran is experiencing homelessness"
```

Comment:
Suggesting a slight alteration to the comment just to be in line with the other ones in the schema
```suggestion
    t.boolean "homelessness", default: false, null: false, comment: "Indicates whether or not a veteran is experiencing homelessness"
```

So based on these changes, we're creating a `boolean` field, using `default: false` so that newly created appeals will be set to `false` unless a `true` value is accepted, and `null: false` means we won't allow for a `null` value to be accepted, is that accurate @jtsangVA?

Reply:
Yep, that's correct @ajspotts